src/decorators/tooltip/tooltip.js

```diff
@@ -0,0 +1,73 @@
+/*jshint unused:false */
+/*jshint -W025 */
+
+function(node, options) {
+	var tooltip, handlers, eventName, enterSection, leaveSection,
+		config = {
+			tagElement: options.tagElement || 'span',
+			className: options.className || 'tooltip',
+			selectorName: options.selectorName || 'tooltip' + Math.floor((Math.random() * 1000) + 1),
+			content: options.content || '',
+			delay: options.delay || 0,
+			show_on: options.show_on || 'small medium large'
+		};
+
+	enterSection = function() {
+		var tooltip_exists = document.getElementById(config.selectorName);
+		if (!tooltip_exists && config.content.length) {
```

Comment:
Style comment: to avoid additional indentation, you can invert the if check and return early, that means you can unindent the below.